app/Http/Controllers/PlatformController.php

```diff
@@ -155,12 +155,8 @@ public function update(PlatformUpdateRequest $request, Platform $platform): Redi
      *
      * @return RedirectResponse
      */
-    public function destroy(Platform $platform): RedirectResponse
+    public function destroy(Request $request, Platform $platform): RedirectResponse
     {
-        if (auth()->user()->cannot('administrate')) {
```

Comment:
Redundant check, it's already covered in middleware with can create platforms. Can we confirm that is good enough? If not, let's move the route to higher middleware.